tests/non-pytest/mypy-ignore-tests/transfer_data.py

```diff
@@ -21,4 +21,4 @@
 tdata.add_filter_rule("*.tgz", type="file")
 # bad values rejected (Literal)
 tdata.add_filter_rule("*.tgz", type="files")  # type: ignore[arg-type]
-tdata.add_filter_rule("*.tgz", method="include")  # type: ignore[arg-type]
+tdata.add_filter_rule("*.tgz", method="unclude")  # type: ignore[arg-type]
```

Comment:
Somehow, this change brings me a great deal of joy.
I know it's just picking some other invalid name, but I can't help but imagine a service supporting "unclude", whatever that means. :joy: